crates/uv/src/commands/python/list.rs

```diff
@@ -167,40 +184,91 @@ pub(crate) async fn list(
         include.push((key, uri));
     }
 
-    // Compute the width of the first column.
-    let width = include
-        .iter()
-        .fold(0usize, |acc, (key, _)| acc.max(key.to_string().len()));
+    match format {
+        PythonListFormat::Json => {
+            let data: Vec<PrintData> = include
+                .iter()
+                .map(|(key, uri)| match uri {
+                    Either::Left(path) => {
+                        let is_symlink = fs_err::symlink_metadata(path).unwrap().is_symlink();
+                        if is_symlink {
```

Comment:
afaict these 3 paths only differ by one field each. I'd like to see this refactored to just compute the 2 special fields and then make one PrintData at the end.

Reply:
Yeah you're right! I'll make that adjustment. Thanks for the pointer.